nilearn/glm/first_level/hemodynamic_models.py

```diff
@@ -160,15 +173,19 @@ def spm_time_derivative(tr, oversampling=50, time_length=32., onset=0.):
           dhrf sampling on the provided grid
 
     """
-    do = .1
-    dhrf = 1. / do * (
-        spm_hrf(tr, oversampling, time_length, onset)
-        - spm_hrf(tr, oversampling, time_length, onset + do)
+    do = 0.1
+    dhrf = (
+        1.0
+        / do
+        * (
+            spm_hrf(tr, oversampling, time_length, onset)
+            - spm_hrf(tr, oversampling, time_length, onset + do)
+        )
     )
```

Comment:
I find those quite hard to read and they probably should be split across several lines, no?